module/Motion/ColorLineTracing.h

```diff
@@ -48,6 +60,26 @@ class ColorLineTracing : public LineTracing {
    */
   void logRunning() override;
 
+  /**
+   * @brief 走行体がコースを外れたときに復帰動作をする
+   */
+  using LineTracing::recover;
+
+  /**
+   * @brief 走行体がコースを脱線しているかを判定をする　返り値がtrueで復帰動作開始
+   */
+  using LineTracing::isErrorState;
+
+  /**
+   * @brief 復帰動作実行のログを取る
+   */
+  using LineTracing::logRunningRecovering;
+
+  /**
+   * @brief 復帰動作終了のログを取る
+   */
+  using LineTracing::logFinishingRecovering;
+
  private:
   static constexpr int JUDGE_COUNT = 3;
   int colorCount = 0;
```

Comment:
```suggestion
  int colorCount = 0;  // コースから外れたと判定するまでにかかる色認識の回数
```
こんな感じで、変数の意味をここに🙏
いい感じの言葉が思いつかない🤔